### DISCUSSION TITLE
[Uno.Themes][Material] Support for missing components?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#7932] [General]
Hi,

What are the plans to support missing components in [Uno.Themes](https://github.com/unoplatform/Uno.Themes)/Material? In particular I'm interested in (sorted by priority):
- TabView

Any insight on futures plans and/or schedule would be highly appreciated.

Thanks

**Comment by limeniye** [comment]
Hm. Im not sure but I think the first element already added
[AutoSuggestBox](https://platform.uno/docs/articles/implemented/windows-ui-xaml-controls-autosuggestbox.html) 

Also you can read information about NumberBox 
[here](https://platform.uno/docs/articles/supported-features.html) 

You can find TabView in the CommunityToolkit.

As I'm sure you can use IconButton from UnoToolkit
but for me esear will be next code:
```xml
<Button>
    <Image Source="image"/>
</Button>
```
because Button inheritance ContentControl. 
You can read about it here
https://github.com/MicrosoftDocs/windows-uwp/blob/docs/hub/apps/design/controls/buttons.md

**Reply by DierkDroth** [reply]
Sorry, this is not about supporting components above in UNO - I understand that they are supported in UNO already.

This is about supporting these components in [Uno.Themes](https://github.com/unoplatform/Uno.Themes)/Material - where they are missing.

**Comment by francoistanguay** [comment]
What is a SymbolButton?

Also, is this assuming that you're happy with the behavior of all these controls and we're only talking about styling here?

**Comment by DierkDroth** [comment]
@francoistanguay thanks for follow up

- sorry I meant SymbolIcon (I fixed my original post)
- I'm not yet there to test the behavior of the components above on all target platforms I have in mind. However, I filed - and will keep filing - issue reports as I came across (e.g. [here ](https://github.com/unoplatform/uno/issues/7778) or [here](https://github.com/unoplatform/uno/issues/7957) or [here](https://github.com/unoplatform/uno/issues/7958))

**Reply by DierkDroth** [reply]
@francoistanguay I removed SymbolIcon from the list above, since I no longer needed it with my current design approach.

**Reply by DierkDroth** [reply]
@francoistanguay I also removed AutoSuggestBox and NumberBox from list above, since I figured it out by myself.
